e2e/e2eutil/cli.go

```diff
@@ -0,0 +1,113 @@
+package e2eutil
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+// Command sends a command line argument to Nomad and returns the unbuffered
+// stdout as a string (or, if there's an error, the stderr)
+func Command(cmd string, args ...string) (string, error) {
+	c := exec.Command(cmd, args...)
+
+	var rawOut bytes.Buffer
+	c.Stdout = &rawOut
+
+	var rawErr bytes.Buffer
+	c.Stderr = &rawErr
+
+	err := c.Run()
+	if err != nil {
+		return rawErr.String(), err
```

Comment:
In practice I've found it useful to just use `CombinedOutput` every time, since it provides more context in error conditions (or output at all, if the executed command doesn't bother with stderr). Not sure if doing so would be super helpful there.

Reply:
Oh, that's a good point. Will fix.

Reply:
Fixed in 13f6674